TRTexture16Importer/Helpers/ColourExtensions.cs

```diff
@@ -146,4 +148,60 @@ public static ushort ToRGB555(this Color c)
 
         return (ushort)(a | r | g | b);
     }
+
+    public static Color ToColor(this TRColour c)
+    {
+        int multiplier = TRConsts.Palette8Multiplier;
+        return Color.FromArgb(c.Red * multiplier, c.Green * multiplier, c.Blue * multiplier);
+    }
```

Comment:
If this is only used by TR1, I think it's worth highlighting it somehow with function names.

Reply:
Makes sense - now renamed. I'm thinking we should only have one colour class eventually and have the IO handle this complexity - so multiplied out on read and divided on write for TR1. But I'll wait until we do more with TR3/4/5 lights as they use colours in different formats. That way we can manipulate colours as we please and the complexity is left until writing back.

It would also need updates to the custom model exports we use for cross-level enemies, so I think handling it later will be best.